Directory.Packages.props

```diff
@@ -43,9 +43,11 @@
     <PackageVersion Include="Microsoft.Extensions.Options" Version="8.0.2" PreserveMajor="true"/>

```

Comment:
Updating LatestSupportedDotNetVersion without updating the Previous DotNet Package Versions item group to target the 9.x package versions will cause the 9.x builds to consume 8.x packages.